packages/graphql-codegen-cli/package.json

```diff
@@ -57,6 +57,7 @@
     "chalk": "^4.1.0",
     "chokidar": "^3.5.2",
     "cosmiconfig": "^7.0.0",
+    "cosmiconfig-typescript-loader": "4.0.0",
```

Comment:
I don't think we need to pin it and can sue a range

Reply:
Agree with @n1ru4l , we can do `^`